src/pages/Credits/index.js

```diff
@@ -0,0 +1,40 @@
+import React, {useEffect} from 'react'
+
+import DefaultLayout from '../../layout/Default'
+import CreditsStyled from './Credits'
+
+
+const Credits = () => { 
+    useEffect(() => {
+        document.title = 'Crédits - Instant Visio'
```

Comment:
Missing translation

Reply:
You can move useEffect to a customHook. Call this customhook in differents layouts. Inject title prop to layout

Reply:
done